sdk/core/core-http/test/policies/bearerTokenAuthenticationPolicyTests.ts

```diff
@@ -52,15 +53,15 @@ describe("BearerTokenAuthenticationPolicy", function() {
     );
   });
 
-  it("refreshes access tokens when they expire", async () => {
+  it("refreshes access tokens when they expire", async function() {
     const now = Date.now();
     const refreshCred1 = new MockRefreshAzureCredential(now);
     const refreshCred2 = new MockRefreshAzureCredential(now + TokenRefreshBufferMs);
     const notRefreshCred1 = new MockRefreshAzureCredential(now + TokenRefreshBufferMs + 5000);
 
     const credentialsToTest: [MockRefreshAzureCredential, number][] = [
-      [refreshCred1, 2],
-      [refreshCred2, 2],
+      [refreshCred1, 1],
+      [refreshCred2, 1],
```

Comment:
What's the difference between 1 and 2 here?

Reply:
number of refreshments have changed because of this PR, since we know use the existing tokens first, and in parallel attempt  to request (?) Please let me know if I'm missing something

Reply:
Ah okay, got it.